src/LedgerLivePlatformSDK/index.ts

```diff
@@ -311,6 +311,10 @@ export default class LedgerLivePlatformSDK {
      * ticker of the currency
      */
     ticker?: string;
+    /**
+     * type of the currency
+     */
+    type?: string;
```

Comment:
I would rather use a CurrencyType `enum` instead of a `string` to further define and restrict allowed values